pkg/commands/compute/init.go

```diff
@@ -332,29 +334,50 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 			}
 
 			if pack.Metadata != nil {
+				clonedFrom := fastly.ToValue(pack.Metadata.ClonedFrom)
+				if serviceVersion > 1 {
+					out.Write([]byte("Service has active versions, not fetching starter kit source"))
```

Comment:
```suggestion
					text.Info(out, "Service has active versions, not fetching starter kit source")
```

You might have to check the output (both with and without the `--verbose` flag) and add preceding or trailing line breaks to make sure this INFO note has some breathing space around it from other output.

If you add line breaks and they're the same regardless of `--verbose` then just add them inline to the string message (e.g. `"\nService has active versions, not fetching starter kit source\n\n"`). Otherwise, if the line break is dependant on `--verbose`, then use something like...

```go
if c.Globals.Verbose() {
    text.Break(out)
}
```

Reply:
Done, I went with fewer branches/more consistent since this feels important for expectation setting—if you run this against an existing service you aren't going to get source